src/Nethermind/Nethermind.Network/Config/INetworkConfig.cs

```diff
@@ -92,12 +100,20 @@ public interface INetworkConfig : IConfig
     [ConfigItem(DefaultValue = "2000", HiddenFromDocs = true, Description = "[TECHNICAL] Outgoing connection timeout in ms. Default is 2 seconds.")]
     int ConnectTimeoutMs { get; set; }
 
-    [ConfigItem(DefaultValue = "1", HiddenFromDocs = true, Description = "[TECHNICAL] Num of thread in final processing of network packet. Set to more than 1 if you have very fast internet.")]
+    [ConfigItem(DefaultValue = "1", HiddenFromDocs = true,
+        Description = "[TECHNICAL] Num of thread in final processing of network packet. Set to more than 1 if you have very fast internet.")]
     int ProcessingThreadCount { get; set; }
 
-    [ConfigItem(DefaultValue = null, HiddenFromDocs = true, Description = "[TECHNICAL] Only allow peer with clientId matching this regex. Useful for testing. eg: 'besu' to only connect to BeSU")]
+    [ConfigItem(DefaultValue = null, HiddenFromDocs = true,
+        Description = "[TECHNICAL] Only allow peer with clientId matching this regex. Useful for testing. eg: 'besu' to only connect to BeSU")]
     string? ClientIdMatcher { get; set; }
 
     [ConfigItem(DefaultValue = "false", HiddenFromDocs = true, Description = "[TECHNICAL] Disable feeding ENR DNS records to discv4 table")]
     bool DisableDiscV4DnsFeeder { get; set; }
+
+    [ConfigItem(DefaultValue = "0", HiddenFromDocs = true, Description = $"[TECHNICAL] Limit the number of bytes sent per second as part of P2P messages. Default is 0 (unlimited)")]
+    int PacketSenderThrottlingBytesPerSecond { get; set; }
```

Comment:
PacketSenderThrottlingBytesPerSecond -> MaxSentBytesPerSecond
ProtocolHandlerThrottlingBytesPerSecond -> MaxReceivedBytesPerSecond
What do you think about this naming?